src/networking.c

```diff
@@ -962,7 +962,7 @@ void addReplyHumanLongDouble(client *c, long double d) {
 
 /* Add a long long as integer reply or bulk len / multi bulk count.
  * Basically this is used to output <prefix><long long><crlf>. */
-void addReplyLongLongWithPrefix(client *c, long long ll, char prefix) {
+void _addReplyLongLongWithPrefix(client *c, long long ll, char prefix) {
```

Comment:
```suggestion
static void _addReplyLongLongWithPrefix(client *c, long long ll, char prefix) {
```

Reply:
Many of the internal functions are lacking the `static` keyword. I've noticed Salvatore removed `static` because the static functions didn't show up in stacktraces (see e.g. 083acbebc85191314840a02a8618f051f49d0319) but that's solved by compiling with `-fno-omit-frame-pointer`. Maybe static isn't important for performance, but it's good for readability.